package.json

```diff
@@ -737,14 +777,140 @@
+                            ]
+                        },
+                        {
+                            "properties": {
+                                "cmakeDebugType": {
+                                    "enum": [
+                                        "configure"
+                                    ]
+                                },
+                                "clean": {
+                                    "type": "boolean",
+                                    "description": "%cmake-tools.debugger.clean.description%",
+                                    "default": false
+                                },
+                                "configureAll": {
+                                    "type": "boolean",
+                                    "description": "%cmake-tools.debugger.configureAll.description%",
+                                    "default": false
+                                },
+                                "dapLog": {
+                                    "type": "string",
+                                    "description": "%cmake-tools.debugger.dapLog.description%",
+                                    "default": ""
+                                }
+                            }
+                        },
+                        {
+                            "properties": {
+                                "cmakeDebugType": {
+                                    "enum": [
+                                        "external"
+                                    ]
+                                }
+                            },
+                            "required": [
+                                "pipeName"
+                            ]
+                        }
+                    ]
+                }
+            },
+            "initialConfigurations": [],
+            "configurationSnippets": [
+                {
+                    "label": "%cmake-tools.debugger.configure.snippet.label%",
+                    "description": "%cmake-tools.debugger.configure.snippet.description%",
+                    "body": {
+                        "type": "cmake",
+                        "request": "launch",
+                        "name": "%cmake-tools.debugger.configure.snippet.body.name%",
+                        "cmakeDebugType": "configure",
+                        "clean": false,
+                        "configureAll": false
+                    }
+                },
+                {
+                    "label": "%cmake-tools.debugger.script.snippet.label%",
+                    "description": "%cmake-tools.debugger.script.snippet.description%",
+                    "body": {
+                        "type": "cmake",
+                        "request": "launch",
+                        "name": "%cmake-tools.debugger.script.snippet.body.name%",
+                        "cmakeDebugType": "script",
+                        "scriptPath": "^\"\\${workspaceFolder}/<...>\""
```

Comment:
What's the `^` for? Does this act as a regex? If it's not, we can probably hardcode something like `<script>.cmake` even though it won't exist just to make the example clearer.

Reply:
I don't think so. I copied it from somewhere and it seemed to simply allow us to insert it without resolving the ${workspaceFolder} to an actual value. 

When I created an automatic debug configuration using this, this is what appeared: 
![image](https://github.com/microsoft/vscode-cmake-tools/assets/86264750/219f7617-5118-4e09-89bc-97f53c0436be)

Reply:
@benmcmorran ping for review.

Specifically, https://github.com/microsoft/vscode-cmake-tools/pull/3277#discussion_r1298817459